src/routes/task/task.ts

```diff
@@ -6,7 +6,7 @@ export class Task {
 
     static async getAllFromTaskListId(req: express.Request, res: express.Response) {
         try {
-            const task = await TaskFacade.getAllFromTaskListId(req.params.taskList_id)
+            const task = await TaskFacade.getAllFromTaskListId(req.params.id)
```

Comment:
id is not clear. Use taskId.

Reply:
id is the attribut name in Trello-route, I thought it is better to keep it.